### PR TITLE
first crack at a docs style guide

### PR DESCRIPTION
**Pull request by smart-alek**
This is completely arbitrary, and maybe some folks will disagree with me. Inviting @li3n3 and @ndintenfass to the fight. :)

### CODE REVIEW

**Comment by ndintenfass** [comment]
I added several in-line comments. Consider merging this into CONTRIBUTING.md.

You should add guides on:
* Use of markdown -- encourage use of markdown over HTML and link to reference on markdown in Liquid and use of front matter (or a separate section on Jekyll conventions around front matter, though it looks like there's some of that in the CONTRIBUTING.md, which makes me think we should really combine this into that.
* Code elements and snippets -- when to use backticks and how to mark code blocks (and how to use `{% raw %}` when doing code blocks where you want to render curly braces -- perhaps a section on 
* Voice - discussion of you/your vs. we/our and such

File: STYLEGUIDE.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+As with code, technical writing benefits from consistent rules. Please follow this guide when adding or editing CircleCI documentation.
```
Comment (smart-alek) [inline STYLEGUIDE.md]: Great point. I'm moving this to CONTRIBUTING until it gets too long.

File: STYLEGUIDE.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+# Structure
+
+- Don't use h1 headers. The title you choose in a doc's frontmatter will automatically be converted into an h1.
+- Add a table of contents to articles with more than 10 h2s.
```
Comment (smart-alek) [inline STYLEGUIDE.md]: Cool, this is already in CONTRIBUTING.md, so this will be solved when I move it there.

File: STYLEGUIDE.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+- Single-spaced, never double-spaced.
+
+# Lists
+
+- Capitalize each list item.
+- Place periods at the end of each list item.
+
+# Bold
+
+- Use for terms that are defined.
```
Comment (smart-alek) [inline STYLEGUIDE.md]: Glossary is one, but also a term that might be defined (and linked) elsewhere, like Jekyll's front matter.

File: STYLEGUIDE.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+- Use for interactive elements, like buttons or links.
```
Comment (smart-alek) [inline STYLEGUIDE.md]: For talking about buttons that are in the app or somewhere in a UI. I'll make this clearer.

File: STYLEGUIDE.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+# Italics
+
+- Use for emphasis.
+
+# Quotes
+
+- Use for non-interactive elements, like section names.
```
Comment (smart-alek) [inline STYLEGUIDE.md]: Cool, might remove this for now. But will add some context around blockquotes.

File: STYLEGUIDE.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
+As with code, technical writing benefits from consistent rules. Please follow this guide when adding or editing CircleCI documentation.
+
+# Structure
+
+- Don't use h1 headers. The title you choose in a doc's frontmatter will automatically be converted into an h1.
+- Add a table of contents to articles with more than 10 h2s.
+
+# Formatting
+
+- Single-spaced, never double-spaced.
```
Comment (smart-alek) [inline STYLEGUIDE.md]: Oh, does the extra whitespace just get stripped out when the Markdown is converted to HTML?

File: STYLEGUIDE.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+# Lists
+
+- Capitalize each list item.
+- Place periods at the end of each list item.
```
Comment (smart-alek) [inline STYLEGUIDE.md]: I agree; this is a bit too contextual to have a recommendation around it.

**Comment by smart-alek** [comment]
Closing this one @ndintenfass in favor of a more robust PR incoming.

File: STYLEGUIDE.md (hunk 6 above)
Comment (li3n3) [inline STYLEGUIDE.md]: On rendering, yeah, if you're talking about two-spaces-after-a-sentence-ending-punctuation.

**Pull request closed without merging**